apps/meteor/app/statistics/server/lib/SAUMonitor.ts

```diff
@@ -185,6 +189,26 @@ export class SAUMonitorClass {
 		// TODO missing an action to perform on dangling sessions (for example remove sessions not closed one month ago)
 	}
 
+	private async _getSearchTerm(session: Omit<ISession, '_id' | '_updatedAt' | 'createdAt'>): Promise<string | undefined> {
+		const user = await Users.findOneById<IUser>(session.userId, {
```

Comment:
this generic is useful to specify only the fields returned by the project, in this case:
```suggestion
		const user = await Users.findOneById<Pick<IUser, 'username' | 'name'>>(session.userId, {
```